### ISSUE TITLE
plot relative drop in effect size by initial sample size etc.

### ISSUE THREAD

**Issue by soodoku** [ecsalomon/TSR---Test-Stats-Replication#1]
Y = (replicated effect size - paper eff. size)/(replicated eff. size)
X = sample size etc.

**Comment by not-the-fish** [comment]
Good idea. This will require some additional coding, as I didn't code which sample sizes came from the critical studies and didn't include paper with single studies. I will give this a shot next week.
